components/add.js

```diff
@@ -195,22 +199,30 @@ export const AddInstancePage = ({ collection }) => {
           });
 
           const errors = response.errors
-            ? response.errors.map((err) => ({
-                description: err.description,
-                keys: err.name
+            ? response.errors.map((err) => {
+                // Surround each err name with ``, and separate by comma
+                const keys = err.name
                   .map((val) => {
                     return `\`${val}\``;
                   })
-                  .join(", "),
-              }))
+                  .join(", ");
+                // Unique identifier for this error object
+                const key = `${keys}${err.description}`;
+                return {
+                  description: err.description,
+                  keys: keys,
+                  key: key,
+                };
+              })
             : [
                 {
                   description:
                     "Error saving new item, ensure the fields are filled out correctly",
                   keys: "Generic Error",
+                  key: "Generic ErrorError saving new item, ensure the fields are filled out correctly",
```

Comment:
Hm yeah the key here is what I'm using to uniquely identify each error object, and it's just a concatenation of the keys and description. I can see that that's confusing in this default case. What do you recommend instead?

Reply:
I’m not sure what it’s supposed to do. You’re saying “ErrorError” serves a purpose? If it does, it stands of very high chance of getting accidentally deleted in a future edit because it looks like a mistake.

Reply:
yeah sorry, the text "ErrorError" itself doesn't serve a purpose. Maybe I can make it clearer in the code.